.github/workflows/ci.yml

```
@@ -39,7 +36,7 @@ jobs:
       run: tox
 
     - name: Run coverage
-      if: matrix.python-version == '3.8' && matrix.toxenv == 'django22-celery44-drf312'
+      if: matrix.python-version == '3.8' && matrix.toxenv == 'django32-celery44-drf312'
```

We should run coverage against latest celery version now. 
```suggestion
      if: matrix.python-version == '3.8' && matrix.toxenv == 'django32-celery50-drf312'
```